src/index.ts

```diff
@@ -1,16 +1,15 @@
-const express = require('express')
-const dotenv = require('dotenv')
+require("dotenv").config();
+const express = require("express");
 
-dotenv.config()
+const app = express();
```

Comment:
@domoritz I am keeping out semi-colons because it looks clean. What do you say?

Reply:
Yep. Prettier should add them automatically.

Reply:
@domoritz I have added a config which doesn't let eslint add them. Shall I remove it? I prefer no semi-colons though :sweat_smile: .